### ISSUE TITLE
[feature request] Keep cursor on previously selected entry for counsel-find-file

### ISSUE THREAD

**Issue by rieje** [abo-abo/swiper#2007]
When navigating the file hierarchy using `counsel-find-file` (I believe using `counsel-yank-directory` and `counsel-up-directory` bind to `C-l` and `C-h` respectively?), it would be nice to have an option to allow `counsel-up-directory` to maintain the previously selected entry.

In other words, have an option where going back up a directory will have the currently selected file/directory be on the directory that was just selected, to allow for much easier task of finding a file. Currently, going back up the directory will always have the selection be on `./`, which makes finding files in subdirectories a chore as you would always have to navigate down to the next directory every time you go back up a directory to continue searching in another sibling directory.

**Comment by abo-abo** [comment]
`counsel-up-directory` already does this. Verified with `make plain`. Please check your config.

**Comment by manuel-uberti** [comment]
I cannot reproduce this either. @rieje, is this still a problem for you?

**Comment by manuel-uberti** [comment]
@abo-abo I edited the previous comment with a recipe.

**Comment by abo-abo** [comment]
@manuel-uberti You should use `C-<backspace>` in place of `<backspace>`:

- `C-<backspace>` is bound to `counsel-up-directory`
- `<backspace>` is bound to `ivy-backward-delete-char`

**Comment by manuel-uberti** [comment]
You're right, and it works as expected. @rieje can you confirm it works for you too?
